docs-compose/overwatch_docs/content/EnvironmentSetup/azure.md

```diff
@@ -77,6 +77,18 @@ situation like this, the minimum autoscaling compute size should approximately e
 minimize waste.  
 {{% /notice %}}
 
+Once created, get the connection string from the SAS Policy for the Event Hub, find the following path in the Azure portal below.
+
+eh-namespace --> eventhub --> shared access policies --> Connection String-primary key
+
+The connection string should begin with `Endpoing=sb://`. Note that the policy only needs the Listen permission
```

Comment:
Please change "Endpoing" to Endpoint -- see [Issue 232](https://github.com/databrickslabs/overwatch/issues/232)